Controller/FormController.php

```diff
@@ -188,13 +189,35 @@ public function cgetTemplateAction(Request $request)
         return $this->render(
             $this->getBundleName() . ':' . $this->getListName() . ':template.html.twig',
             [
-                'types' => Dynamic::$TYPES,
+                'types' => $this->getSortedTypes(Dynamic::$TYPES),
                 'widths' => $widths,
             ]
         );
     }
 
     /**
+     * @param $types
+     *
+     * @return array
+     */
+    public function getSortedTypes($types = [])
+    {
+        /** @var Translator $translator */
+        $translator = $this->get('translator');
+        $prefix = 'l91_sulu_form.type.';
+        $sortedTypes = [];
+
+        foreach ($types as $type) {
+            $translation = $translator->trans($prefix . $type, array(), null, 'de');
+            $sortedTypes[$translation] = $type;
```

Comment:
pls add `$translation . $key` to avoid that when the name of 2 types are the same it will not replaced...